doc/source/users_guide/testing.rst

```diff
@@ -220,19 +220,19 @@ To run two tests::
 
 To run a test suite::
 
-  ./create_test acme_developer
+  ./create_test e3sm_developer
 
 To run a test suite excluding a specific test::
 
-  ./create_test acme_developer ^SMS.f19_f19.A
+  ./create_test e3sm_developer ^SMS.f19_f19.A
 
 See create_test -h for the full list of options
 
 Interpreting test output is pretty easy, looking at an example::
 
   % ./create_test SMS.f19_f19.A
 
-  Creating test directory /home/jgfouca/acme/scratch/SMS.f19_f19.A.melvin_gnu.20170504_163152_31aahy
+  Creating test directory /home/jgfouca/e3sm/scratch/SMS.f19_f19.A.melvin_gnu.20170504_163152_31aahy
```

Comment:
This was one of the instances that was ok to leave as acme.  But its just an example and it doesn't matter if the dir path is real or not so don't redo it.